compiler/dyno/include/chpl/uast/prim-ops-list.h

```diff
@@ -257,6 +257,8 @@ PRIMITIVE_G(STRING_CONCAT, "string_concat")
 PRIMITIVE_G(STRING_LENGTH_BYTES, "string_length_bytes")
 PRIMITIVE_G(STRING_LENGTH_CODEPOINTS, "string_length_codepoints")
 PRIMITIVE_G(ASCII, "ascii")
+PRIMITIVE_G(STRING_ITEM, "string item")
+PRIMITIVE_G(BYTES_ITEM, "bytes item")
```

Comment:
recommend using `PRIMITIVE_R` for these to indicate the primitive should not exist after resolution (which makes sense here because you only use them for `param`).

Reply:
Well, I learned something today. Or remembered... I don't want to believe that I didn't know this.